### ISSUE TITLE
dust model test

### ISSUE THREAD

**Issue by changhoonhahn** [changhoonhahn/gqp_mc#23]
At the moment we're using a simple Calzetti dust attenuation. We need to test whether this biases our parameter constraints by comparing to a model with a more flexible dust model. The more flexible dust model has already been implemented under `model = fsps_complexdust`.

**Comment by changhoonhahn** [comment]
@kgb0255 After you've verified that you can reproduce `nb/DESI_speculator_comparison.ipynb`. Could you re-run the MCMCs for `fsps` and `fsps_complexdust` for the first 10 galaxies?

**Comment by kgb0255** [comment]
@changhoonhahn  Yes. Just for the record, I have `fsps` version 0.3.0 and I git pulled the updated repo. I believe these two are the only updates in our pipeline. I queued the chains for both `fsps` and `fsps_complexdust`.

**Comment by changhoonhahn** [comment]
@kgb0255 Could you post the figure you had in your DESI research forum talk? The one that showed the inferred properties with simple dust vs complex dust.

**Comment by kgb0255** [comment]
Yup!
Simple dust model

<img width="1278" alt="Screen Shot 2020-10-22 at 10 03 56 AM" src="https://user-images.githubusercontent.com/57269223/96906018-1529d600-144e-11eb-8312-7cbf7ba3b45c.png">

Complex dust model

<img width="1278" alt="Screen Shot 2020-10-22 at 10 04 37 AM" src="https://user-images.githubusercontent.com/57269223/96906039-1bb84d80-144e-11eb-9da2-8aa272c05025.png">
